features/step_definitions/aspects_steps.rb

```diff
@@ -88,6 +88,21 @@ def aspect_dropdown_visible?
   end
 end
 
+When /^I drag "([^"]*)" (up|down) (\d+) pixels?$/ do |aspect_name, direction, distance|
+  distance = distance.to_i * -1 if direction == 'up'
+  page.execute_script %{
+    $.when(
+      $.getScript("/assets/jquery-simulate/jquery.simulate.js"),
+      $.getScript("/assets/jquery-simulate-ext/src/jquery.simulate.ext.js"),
+      $.getScript("/assets/jquery-simulate-ext/src/jquery.simulate.drag-n-drop.js"),
+      $.Deferred(function( deferred ){ $(deferred.resolve); })
```

Comment:
So what happens if the step is called multiple times without reloading the page?